ui/.eslintrc.js

```diff
@@ -40,7 +40,14 @@ module.exports = {
         // Add Cypress-specific lint rules, globals and Cypress plugin
         // See https://github.com/cypress-io/eslint-plugin-cypress#rules
         'plugin:cypress/recommended'
-      ]
+      ],
+
+      rules: {
+        'cypress/no-unnecessary-waiting': 'off',
+        // It doesn't even allow .then(), it is too restrictive. It's quite new, needs time to mature.
+        'cypress/unsafe-to-chain-command': 'off',
```

Comment:
This is probably no longer needed as #15782 got merged. @freddy38510 please confirm if the changes you've made were about this rule.

Reply:
@yusufkandemir, yes that was totally about this eslint rule.

Reply:
@freddy38510 thanks for confirming.

I removed the rule overrides, which caused other places to fail. I also addressed those places. Everything should be fine now.